homeassistant/components/enphase_envoy/__init__.py

```diff
@@ -20,6 +20,10 @@
 )
 from .coordinator import EnphaseConfigEntry, EnphaseUpdateCoordinator
 
+# coordinator is used to get all data centralized.
+# Set parallel updates default to 1 for all platform action updates.
+PARALLEL_UPDATES = 1
```

Comment:
If I'm not mistaken these has to go into the platform files to have any effect. Setting it here will have no effect.

Reply:
Found it in \_init\_.py for iaqualink when scanning for examples. But I now see that iaqualink has it in the platform files as well. Hoped for a one spot default, but sounds like not and need to add it to the platforms files instead.